src/d3a/is_installed.py

```diff
@@ -0,0 +1,7 @@
+
+def is_installed(module_name):
```

Comment:
I couldn't find its usage.

Reply:
Weird, I thought I deleted it. You are right, it can be removed.